.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: Template for reporting bugs
+title: ''
+labels: bug
+assignees: ''
+
+---
+
```

Comment:
I think we are missing a **Description** section, it's basically a summary of the bug. It could be something like:

> ## Bug description
A clear and concise description of the bug.